### PR TITLE
Update mandb from 2.8.4 to 2.8.6

### PR DESCRIPTION
**Pull request by cstrouse**
Update the package, DRY up the sed rewrites, and remove dependencies in core.

Tested on ARM.

Yes, there's an update to 2.8.7 available but the files on the mirrors aren't downloading so this will do for now.

### CODE REVIEW

File: packages/mandb.rb [hunk 1]
```diff
@@ -35,22 +22,25 @@ def self.build
...
+      'man/man1/whatis.man1',
+      'man/man1/apropos.man1',
+      'man/man1/man.man1',
+      'man/man8/accessdb.man8',
+      'man/man8/mandb.man8',
+      'tools/chconfig'
+    ].each { |file|
+      system "sed -i 's,/var/cache/man,#{CREW_PREFIX}/cache/man,g' #{file}"
+    }
     system "sed -i 's,/var/cache/man,#{CREW_PREFIX}/cache/man,g' tools/chconfig"
```
Comment (uberhacker) [inline packages/mandb.rb]: Redundant line?

File: packages/mandb.rb [hunk 2]
```diff
@@ -35,22 +22,25 @@ def self.build
...
+      'manual/files.me',
+      'man/man1/whatis.man1',
+      'man/man1/apropos.man1',
+      'man/man1/man.man1',
+      'man/man8/accessdb.man8',
+      'man/man8/mandb.man8',
+      'tools/chconfig'
+    ].each { |file|
+      system "sed -i 's,/var/cache/man,#{CREW_PREFIX}/cache/man,g' #{file}"
+    }
```
Comment (uberhacker) [inline packages/mandb.rb]: Nice rubyizing!

File: packages/mandb.rb [hunk 3]
```diff
@@ -35,22 +22,25 @@ def self.build
...
     system "sed -i 's,/var/cache/man,#{CREW_PREFIX}/cache/man,g' tools/chconfig"
-    system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
+    system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
```
Comment (uberhacker) [inline packages/mandb.rb]: It might be worth adding this package to "core" since it goes without saying it is vital to any linux cli user.

Comment (uberhacker) [reply packages/mandb.rb]: Oh wait...it's already in core. :)

File: packages/mandb.rb (hunk 1 above)
Comment (cstrouse) [reply packages/mandb.rb]: Oh, that's actually a mistake. There *should* be a line after the block but not that one.

File: packages/mandb.rb [hunk 4]
```diff
@@ -35,22 +22,25 @@ def self.build
...
+      'man/man1/whatis.man1',
+      'man/man1/apropos.man1',
+      'man/man1/man.man1',
+      'man/man8/accessdb.man8',
+      'man/man8/mandb.man8',
+      'tools/chconfig'
+    ].each { |file|
+      system "sed -i 's,/var/cache/man,#{CREW_PREFIX}/cache/man,g' #{file}"
+    }
+    system "sed -i 's,/var/cache/man,#{CREW_PREFIX}/share/man,g' tools/chconfig"
```
Comment (uberhacker) [inline packages/mandb.rb]: `system "sed -i 's,/var/share/man,#{CREW_PREFIX}/share/man,g' tools/chconfig"`?

Comment (cstrouse) [reply packages/mandb.rb]: Good catch. Should be /usr/share/man -> #{CREW_PREFIX}/share/man.

**Pull request merged**